build.gradle

```diff
@@ -68,3 +68,19 @@ idea {
         sourceDirs += file('src/api/java')
     }
 }
+
+test {
+    workingDir = {minecraft.runDir + "/test"} // This can be set to whatever you prefer
+
+    mkdir workingDir
+    file("$workingDir/eula.txt").text = "eula=true"
```

Comment:
I would hide this after a config option in barometer, something like `barometer.acceptEula()`.